janusgraph-core/src/main/java/org/janusgraph/core/attribute/Geoshape.java

```diff
@@ -312,7 +313,7 @@ public static final Geoshape box(final double southWestLatitude, final double so
      */
     public static final Geoshape line(List<double[]> coordinates) {
         Preconditions.checkArgument(coordinates.size() >= 2, "Too few coordinate pairs provided");
-        List<com.spatial4j.core.shape.Point> points = new ArrayList<>();
+        List<org.locationtech.spatial4j.shape.Point> points = new ArrayList<>();
```

Comment:
is there another Point.class in scope? I didn't see a second explicitly called out in the imports. If so, can we just rely on the imported Point?